torchdata/datapipes/iter/util/common.py

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from torchdata.datapipes.iter import Mapper
+
+
+class MapTemplateIterDataPipe(Mapper):
+    def __init__(self, source_datapipe, input_col=None, output_col=None) -> None:
```

Comment:
1. Let's add a docstring for the purpose of this class.
2. Add the class `MapTemplate` to RST documnetation file as well.

Reply:
I will update DP contributors doc and add docstrings